doc/source/docker/run-quickstart-examples-docker-compose.rst

```diff
@@ -0,0 +1,114 @@
+Run Flower Quickstart Examples with Docker Compose
+==================================================
+
+Flower provides a set of `quickstart examples <https://github.com/adap/flower/tree/main/examples>`_
+to help you get started with the framework.
+
+These examples are designed to demonstrate the capabilities of Flower and can be run
+using Docker Compose.
```

Comment:
```suggestion
These examples are designed to demonstrate the capabilities of Flower and by default run using the Simulation Engine. This guide demonstrates how to run them using Flower's Deployment Engine via through Docker Compose.
```

Reply:
I thought of rewriting it slightly. Probably we can iterate more. Also, should this be a different paragraph from the first sentence in the doc page? or should we join them ?

Reply:
> I thought of rewriting it slightly. P
That's sounds better! 
I think we can join them. I have no strong preference.